Automattic/wp-calypso
# My Home: Adding Quick Start video card to be featured in Secondary

This PR adds a new card that features a Quick Start Session -style video that introduces new users to the admin interface and the block editor.

This PR also updates the InlineSupportLink component to support a "hero image" to add an option of featuring a screenshot instead of copy.

Here's a screenshot of the Quick Start card shown in the Site Setup view:
<img width="1085" alt="Screen Shot 2020-07-27 at 12 04 08 PM" src="https://user-images.githubusercontent.com/35781181/88568431-024a6580-d007-11ea-95c6-d1ee15f2c81a.png">

And here's a screenshot showing the video in the support link modal:
<img width="1204" alt="Screen Shot 2020-07-27 at 12 04 18 PM" src="https://user-images.githubusercontent.com/35781181/88568497-13937200-d007-11ea-91e2-db443df021e7.png">

Since the video is in English, only people in the US will see this card (handled by the server test allocation, and I'll try and update it to a language check rather than a geo check).

The companion server patch is D46859-code.
More context is at pbBQWj-dh-p2.

This is setup as a 50/50 test allocated by the server, with half of the users seeing the Stats card as they do today, and half seeing this new card. 

Test details:
* Name: my_home_quick_start_video
* Variation names: 'stats' and 'quick-start'

CC @lauac @anneforbush @ballio2000

#### Testing instructions

* Start this branch on your machine
* Point API calls to your sandbox
* Patch your sandbox with D46859
* Confirm that API calls are hitting your sandbox 
* Load My Home with a site that hasn't completed site setup (and loads the Site Setup view)
* Check your cookies to confirm that you're allocated to the 'quick-start' variant
* Confirm that the Quick Start card loads, and that inline support modal works properly when you click the video image.
